plugins/image-prioritizer/class-image-prioritizer-img-tag-visitor.php

```diff
@@ -19,6 +19,11 @@
  */
 final class Image_Prioritizer_Img_Tag_Visitor extends Image_Prioritizer_Tag_Visitor {
 
+	/**
+	 * The ID used the register the class
+	 */
+	public const ID = 'image-prioritizer-img';
```

Comment:
This would also make sense to add to `Embed_Optimizer_Tag_Visitor`.

Reply:
done

Reply:
Oh yeah, I don't see any change to the `Embed_Optimizer_Tag_Visitor` class in the Embed Optimizer plugin?

Reply:
@westonruter I am not sure what is going on here you might need to take this over